src/ajax/check_hgvs_dialogue.php

```diff
@@ -35,17 +35,30 @@
 header('Content-type: text/javascript; charset=UTF-8');
 
 
+// Check whether all required input was given.
+if (!(isset($_REQUEST['var']) && isset($_REQUEST['action'])
+    && isset($_REQUEST['fieldName']) && isset($_REQUEST['refSeqInfo']))) {
+    // If any of these variables are missing, we cannot correctly
+    //  perform any checks, so we will exit the script.
+    exit;
+}
+if (!isset($_REQUEST['transcripts'])) {
+    // Let's assume the transcripts were empty if this variable was not set.
+    $_REQUEST['transcripts'] = '';
+}
+
+
 // Retrieving the transcripts to map to.
 // We are using REQUEST and not GET or POST, because the
 //  input of this script can be both GET and POST.
-$aTranscripts = (empty($_REQUEST['transcripts'])? array() : explode('|', urldecode($_REQUEST['transcripts'])));
+$aTranscripts = (empty($_REQUEST['transcripts'])? array() : explode('|', $_REQUEST['transcripts']));
 
 // Retrieving the name of the input field.
-$sFieldName = htmlspecialchars(urldecode($_REQUEST['fieldName']));
+$sFieldName = htmlspecialchars($_REQUEST['fieldName']);
 
 
 
-if (!($_REQUEST['var'])) {
+if (!empty($_REQUEST['var'])) {
     // If the variant is empty, we can simply close the script.
     exit;
 }
```

Comment:
This kills the script entirely :laughing:

Reply:
By the way, this block can now be removed because you already check for an empty variable on line 39.

Reply:
Whoops ':), the code block has been removed.